content/post/practical_ml_2.md

```diff
@@ -0,0 +1,55 @@
+---
+author: Ilan Man
+title: "More Tips for Practical Data Science"
+cover: ""
+publishDate: TBD
+draft: false
+tags:
+ - datascience
+---
+
+This is the second installment of my ["Tips on Practical Data Science"](https://www.locallyoptimistic.com/post/practical_ml/). In today's post we are going to discuss implications of a changing data distribution, some practical technical considerations, and the value of presenting your work. 
```

Comment:
I think this content is great, but I bet that more people will read and benefit from it if we are a little more descriptive in the title, intro, and section headers about the theme.  What about something like 

"Keeping your team and models fresh as your data world evolves"
 
* Training data != Future data - it's normal for distributions and relationships to change over time
* Deploy != Done - planning for model upkeep is core to effectiveness
* Knowledge does not diffuse automatically - learning is happening all around you, but you need to invest in keeping your team update to a changing world of tools and mental models.

I think you've got a meaningful theme here, and we should communicate it rather than hope our brand new readership is tuned into the series of posts

Reply:
^^ I like this idea of tying these concepts together with this theme!

Reply:
@swiftsam Are you suggesting that I remove the reference to the "part 1" of the series and intro this post as if it lives on its own?

If so, then would I tie this post back to the part 1 below the fold?

Happy to expand the intro and provide more context - I wanted to be concise and get to the meat of the post, but I hear you that a more descriptive intro could help with new readers who won't necessarily click the links leading them back to part 1.